lambda-template-nodejs/index.js

```diff
@@ -21,28 +21,17 @@
 **/
 
 const config = require('./components/config.js'); //Import the environment data.
-const secretHandlerModule = require("./components/secret-handler.js"); //Import the secret-handler module.
 const logger = require("./components/logger.js"); //Import the logging module.
+const responseObj = require("./components/response.js"); //Import the response module.
 
```

Comment:
Include error-handler that you have added as a component and also add a (commented out) code snippet as to how to use it

Reply:
Done.